docs/stdlib/bool.rst

```diff
@@ -52,10 +51,9 @@ Booleans
         db> select (False, false, FALSE);
         {(false, false, false)}
 
-    A boolean value may arise as a result of a :ref:`logical
-    <ref_std_logical>` or :eql:op:`comparison <eq>`
-    operations as well as :eql:op:`in`
-    and :eql:op:`not in <in>`:
+    Boolean values may arise as a result of :ref:`logical <ref_std_logical>`
+    or :eql:op`comparison <eq>` checks, as well as :eql:op`in` and
+    :eql:op`not in <in>` operations:
```

Comment:
```suggestion
```

Reply:
I feel like we still need to inform people about negated `in` operations. Maybe we could code-format this as "`not in` ..." what do you think @raddevon ?